assignment8.txt

```diff
@@ -0,0 +1,11 @@
+1. Compare and contrast fprintf() and printf(). Be thorough - include details on input, output, and details of the functions. 
+	Although printf and fprintf both write text to wherever the standard ouput stream is pointing, their syntax is different, as is when you would use them. fprintf () prints to an ouput file, and its syntax is 'int fprintf (FILE *fptr). Printf is for printing a character stream of data on stdout console. Its syntax is int printf (const char* str...).
+	
+2. What, if anything, is the difference between opening a file in "rw" mode and opening it in "wr" mode? What is the difference between opening a file in "w" mode and opening it in "a" mode? Give an example of when you would use each.
+	There is no such thing as opening a file in wr mode, but when you open it in rw mode, you are replacing a word with something else in your file. When you open a file in 'w' mode, you are opening a file to write in it. When you open a file in 'a' mode, it means that you are appending it (adding as a supplement).
```

Comment:
there is such thing as "wr" mode! “wr” writes, then reads, so the file will be cleared first for writing. “rw” reads, then writes, so the file will not be cleared until the first time you  output to the file.